### DISCUSSION TITLE
New ORTUR Laser Master 2 with Alarm

### DISCUSSION THREAD

**Discussion by Anto217** [arkypita/LaserGRBL#1153] [Q&A]
Dear All, 

Hope you are doing great. 

Wanted some help with you. I am a beginner. Bought my Laser Master  2 (Ortur), got it, assembled it, dowloaded Laser GRBLv3.8.0 and the custom buttons. 

The software communicates with the hardware, but when it connects, the right hand corner status says "Alarm" and the left hand corner says {MSG: Check Limits}

Also, One of the parts came misassembled. Please guide me!! Do you know what the problem is? and how to fix it?

![wrong part](https://user-images.githubusercontent.com/73799597/97816376-931e8600-1c73-11eb-95da-d27d2b1f4bde.jpeg)
![ortur laser master 2](https://user-images.githubusercontent.com/73799597/97816382-9e71b180-1c73-11eb-8e00-862fabdfe39a.jpeg)

**Comment by Anto217** [comment]
![error in software](https://user-images.githubusercontent.com/73799597/97816500-7e8ebd80-1c74-11eb-8643-efd83dcfcdac.jpg)

**Comment by arkypita** [comment]
LaserGRBL is a free software downloadable from the internet, not tied to a specific product.
I suggest you to contact the specialized assistance of your engraver.

For ORTUR brand:
- [Ortur user group (facebook community)](https://www.facebook.com/groups/orturusergroup)
- [Ortur customer support](https://ortur.tech/support/)

Best regards
Arkypita, developer of LaserGRBL
